assets/dev/js/editor/components/selection/manager.js

```diff
@@ -35,6 +35,10 @@ export default class Manager extends elementorModules.editor.utils.Module {
 			get( target, prop ) {
 				if ( [ 'add', 'remove' ].includes( prop ) ) {
 					return ( ...args ) => {
+						if ( 'edit' !== elementor.channels.dataEditMode.request( 'activeMode' ) ) {
+							return;
+						}
```

Comment:
Check if it need to be inside the `add` and `remove`

Reply:
DOne